release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2855,6 +2855,73 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.4
+[id="ocp-4-17-4_{context}"]
+=== RHSA-2024:8981 - {product-title} {product-version}.4 bug fix, and security update advisory
+
+Issued: 13 November 2024
+
+{product-title} release {product-version}.4, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:8981[RHSA-2024:8981] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:8984[RHSA-2024:8984] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.4 --pullspecs
+----
+
+[id="ocp-4-17-4-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-17-4-enhancements-nodes-pods-short-term-auth_{context}"]
+===== Authenticating customer workloads with {gcp-wid-short}
+
+With this release, applications in customer workloads on {product-title} clusters that use {gcp-wid-first} can authenticate by using {gcp-wid-short}.
+
+To use this authentication method with your applications, you must complete configuration steps on the cloud provider console and your {product-title} cluster.
+
+//For more information, see xref:../nodes/pods/nodes-pods-short-term-auth.adoc#nodes-pods-short-term-auth-configuring-gcp_nodes-pods-short-term-auth[Configuring {gcp-wid-short} authentication for applications on {gcp-short}].
```

Comment:
Should this be commented out?

Reply:
Yes, sorry. https://github.com/openshift/openshift-docs/pull/83682 will comment this link in.